doc/source/security/encrypting-system-tablespace.rst

```diff
@@ -0,0 +1,55 @@
+.. _encrypting-system-tablespace:
+
+==================================================================
+Encrypting the System Tablespace
+==================================================================
+
+:Availability: This feature is **Experimental** quality.
```

Comment:
This was GAed in 8.0.16-7 - so no longer Experimental.

Reply:
Removed Experimental tag